modules/my_health/spec/request/v1/message_drafts_request_spec.rb

```diff
@@ -13,120 +13,168 @@
   let(:created_draft_reply_id) { 674_944 }
   let(:draft) { attributes_for(:message, body: 'Body 1', subject: 'Subject 1') }
   let(:params) { draft.slice(:category, :subject, :body, :recipient_id) }
+  let(:mhv_account_type) { 'Premium' }
   let(:va_patient) { true }
   let(:current_user) { build(:user, :mhv, va_patient:, mhv_account_type:) }
   let(:inflection_header) { { 'X-Key-Inflection' => 'camel' } }
 
   before do
-    allow(SM::Client).to receive(:new).and_return(authenticated_client)
     sign_in_as(current_user)
   end
 
-  context 'Basic User' do
-    let(:mhv_account_type) { 'Basic' }
-
-    before { post '/my_health/v1/messaging/message_drafts', params: }
-
-    include_examples 'for user account level', message: 'You do not have access to messaging'
-    include_examples 'for non va patient user', authorized: false, message: 'You do not have access to messaging'
-  end
+  context 'when sm session policy is enabled' do
+    before do
+      Flipper.enable(:mhv_sm_session_policy)
```

Comment:
can you mock the response instead of updating the actual Flipper? That way you won't have to deal with potential flaky specs:
`allow(Flipper).to receive(:enabled?).with(mhv_sm_session_policy).and_return(true)`

Reply:
Thanks for the suggestion, I will check that out. Would this be just in place of `Flipper.enable(:mhv_sm_session_policy)` and would I need to remove `after do` where it's resetting the flipper back?

Reply:
Yes, exactly that

Reply:
That way when specs are split and reorganized you won't hit cases where flipper was enabled by a different spec running concurrently when it meant to be disabled, etc

Reply:
I think these requested changes might not be necessary as of `flipper@1.2.0`. From the docs:

>Since 1.2.0, Flipper will default to the Memory adapter when running tests in a Rails app and reset it before each test. This means each test is run with a fresh instance of Flipper and all features disabled. We find this the best route, but your application may have different needs. If you want some features to always be enabled, you can enable them in your test setup.

https://www.flippercloud.io/docs/testing

Reply:
@bosawt let me know if you still think I should adjust specs considering input by @dickdavis as I have not looked back into changing this yet. Unfortunately, my opinion has no weight here as i'm not an expert in this framework

Reply:
I tested this just to see
<img width="1095" alt="image" src="https://github.com/department-of-veterans-affairs/vets-api/assets/42326799/adfe03c8-f7ab-4a13-bf03-7f7af92231eb">

So the Flipper value is by default set to `enabled` (since it's set to `enabled_in_development`), and I ran the specs so the first spec sets it to `disabled`, and then the second spec runs, but you can see it's reset to `enabled` regardless (even if I remove the `Flipper.enable` component). So I think what Dick linked does represent the current functionality

Reply:
So yeah, you technically shouldn't need any changes. You probably don't need the `after do` cleanup either but at worst it's a no-op so I'll leave that up to you